lib/mindwendel_web/templates/static_page/kits_home.html.leex

```diff
@@ -0,0 +1,48 @@
+<link phx-track-static rel="stylesheet" href="<%= Routes.static_path(@conn, "/css/kits_home.css") %>"/>
+<div id="wrapper">
+
+  <div id="inner">
+    <img src="<%= Routes.static_path(@conn, "/images/mindwendel.svg") %>" alt="mindwendel" style="width: 100%;">
```

Comment:
This image is not available. Leads to broken landing page.

<img width="1904" alt="image" src="https://user-images.githubusercontent.com/592424/108682405-88bd1c00-74f0-11eb-95df-4b290bd425a0.png">

Reply:
thanks, the file was in the wrong folder and was ignored by git